integration/rpcClient.go

```diff
@@ -0,0 +1,67 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/io"
+	"github.com/docker/go-connections/nat"
+	"github.com/stretchr/testify/assert"
+)
+
+type RPCClient struct {
+	addr string
+	port nat.Port
+}
+
+// NewRPCClient creates new client for RPC communications.
+func NewRPCClient(port nat.Port) *RPCClient {
+	addr := "http://127.0.0.1" + ":" + port.Port()
+	return &RPCClient{port: port, addr: addr}
+}
+
+// SendTX sends transaction.
+func (c *RPCClient) SendTX(tx *transaction.Transaction, t *testing.T) []byte {
+	writer := io.NewBufBinWriter()
+	tx.EncodeBinary(writer.BinWriter)
+	rpc := fmt.Sprintf(`{"jsonrpc": "2.0", "id": 1, "method": "sendrawtransaction", "params": ["%s"]}`, hex.EncodeToString(writer.Bytes()))
```

Comment:
Crafting yet another RPC client just for `sendrawtransaction` doesn't seem right to me.

Reply:
we can't use the same one used in rpc tests since there it's created very weird using handler generated by server.
Since here we're not starting server and not supposed to -- we need another approach.

Reply:
??? We're running RPC clients in `cli` without any issues, I'm not sure how using client makes you run a server, true, they're in the same package at the moment, but nothing requires you running it along with client.

Reply:
now I see which client you mean. I can try to reuse that and add required structures to unmarshal response.
Since in `server_test' we're using different one which requires handler which got from server:
server_test.go:
```func doRPCCall(rpcCall string, handler http.HandlerFunc, t *testing.T) []byte {```

```
rpcServer := NewServer(chain, cfg.ApplicationConfiguration.RPC, server)
handler := http.HandlerFunc(rpcServer.requestHandler)
```